cmd/entrypoints/serve.go

```diff
@@ -87,7 +87,7 @@ func blanketAuthorization(ctx context.Context, req interface{}, _ *grpc.UnarySer
 		return handler(ctx, req)
 	}
 
-	if !identityContext.Scopes().Has(auth.ScopeAll) {
+	if !identityContext.Scopes().Has(auth.ScopeAll) || !identityContext.Scopes().Has("all") {
```

Comment:
can we keep the const as such and add another one ? Probably easier to upstream with that

Reply:
this will actually not work .. it should be && instead of || ?